### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_Visual Studio Code User and Workspace settings_Resize Text: On 200% zooming, keyboard focus is not visible on the hamburger menu.

### ISSUE THREAD

**Issue by Chandan5555** [microsoft/vscode-docs#2652]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
**Environment Details:**
URL: code.visualstudio.com/docs/getstarted/settings
Browser: Microsoft Edge 44.18362.1.0

**NOTE:**
This issue is present in the below pages in the application.
code.visualstudio.com/docs                                                      
code.visualstudio.com/download                                                  
code.visualstudio.com/docs/supporting/faq
code.visualstudio.com/docs/setup/linux
code.visualstudio.com/docs/getstarted/settings
code.visualstudio.com/docs/python/python-tutorial
code.visualstudio.com/updates/v1_30
code.visualstudio.com/docs/editor/whyvscode
code.visualstudio.com/docs/getstarted/tips-and-tricks
code.visualstudio.com/updates/v1_29
code.visualstudio.com/docs/editor/integrated-terminal
code.visualstudio.com/docs/editor/debugging
code.visualstudio.com/docs/editor/versioncontrol

**Repro Steps:**
1.Hit the URL:code.visualstudio.com/docs/getstarted/settings
2.Zoom the page to 200 %.
3.Tab till the hamburger menu and check if the focus is clearly visible or not

**Actual result:**
On 200% zooming, keyboard focus is not visible on the hamburger menu.

**Expected result:**
On 200% zooming, keyboard focus should be clearly visible on the hamburger menu.

**User Impact:**
Keyboard user will find it difficulty to navigate the controls if the focus is not visible on 200% zooming while using keyboard

**MAS Reference:**
MAS2.4.7
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c} 
MAS1.4.4
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Comment by Chandan5555** [comment]
![Bug_2652_focus is not visible in 200%](https://user-images.githubusercontent.com/50437119/57451758-c602dc00-727f-11e9-8744-31f119bd564e.png)

**Comment by Chandan5555** [comment]
![bug_2652_on zooming hemberger button is not clearly visible](https://user-images.githubusercontent.com/50437119/57607999-d8d02600-7589-11e9-8750-300b92ca5cdf.png)

**Comment by Chandan5555** [comment]
![bug_2652_on zooming hemberger button is not clearly visible2](https://user-images.githubusercontent.com/50437119/57608501-ac68d980-758a-11e9-8acf-ca221b5c49ea.png)

**Comment by Chandan5555** [comment]
![Bug_2652_focus is not visible on 200% zooming](https://user-images.githubusercontent.com/50437119/57609211-103fd200-758c-11e9-8e12-aa944aaaaa6f.png)

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
